legal-api/src/legal_api/config.py

```diff
@@ -171,6 +171,10 @@ class _Config():  # pylint: disable=too-few-public-methods
     MRAS_SVC_URL = os.getenv('MRAS_SVC_URL')
     MRAS_SVC_API_KEY = os.getenv('MRAS_SVC_API_KEY')
 
+    # GCP Gotenberg report service
+    GOTENBERG_REPORT_API_AUDIENCE = os.getenv('GOTENBERG_REPORT_API_AUDIENCE', '')
+    GOTENBERG_REPORT_SVC_URL = os.getenv('GOTENBERG_REPORT_SVC_URL', 'http://')
```

Comment:
I've added the following in 1pass "api" vault.

![image](https://github.com/user-attachments/assets/33f53dbe-5ddb-40ac-b3d6-349c3bdfdcfb)